site/docsource/OCaml-call-JS.adoc

```diff
@@ -1268,11 +1268,11 @@ var v2 = {
 Compiler infers the type of `v2` as below:
 [source,ocaml]
 --------------
-val v2 : object
-  method hi : float -> float -> float
-  method say : float -> float
-  method x : unit -> float
-end [@bs]
+val v2 : <
+  hi : float -> float -> float;
+  say : float -> float;
+  x : unit -> float
+> [@bs]
```

Comment:
indeed, it is intentional, note Object method is different from function ( due to `this`),
 the equivalent one would be 
```ocaml
< hi : float -> float -> float [@bs.meth]; > Js.t
```